polyfill/intersection-observer.js

```diff
@@ -368,6 +368,10 @@ IntersectionObserver.prototype._computeTargetAndRootIntersection =
 
   while (!atRoot) {
     var parentRect = null;
+    var parentComputedStyle = window.getComputedStyle(parent);
```

Comment:
This broke things for me on Firefox and Safari (I think it's when the parent is the html element). The version before this change works fine.